raspberry-pi/5/overlay.nix

```diff
@@ -0,0 +1,6 @@
+final: _prev: {
+  linux_rpi5 = final.linux_rpi4.override {
```

Comment:
I'm probably missing something, but what's the motivation behind overriding `linux_rpi4` from `final` and not from `prev`?

Reply:
If `prev` was used instead of `final`, in the case where another overlay modified `linux_rpi4` the resulting behavior would depend on the overlay order, which is likely undesirable.